### PR TITLE
Convert 2017 components to use CivicStoryCard and remove StoryCard

### PR DESCRIPTION
**Pull request by jaronheard**
This addresses #602, and should close out #545.

For reference, here's what the 2017 projects look like now. See #602 for details on why no watermarks and footers.

Screen Shot 2019-06-07 at 14 51 26
Screen Shot 2019-06-07 at 14 50 35
Screen Shot 2019-06-07 at 15 33 45
Screen Shot 2019-06-07 at 15 33 56
Screen Shot 2019-06-07 at 15 34 04

### CODE REVIEW

File: packages/budget/src/components/CardCollection/index.js [hunk 1]
```diff
@@ -19,11 +20,11 @@ const CardCollection = () => (
           decision-making!
         </p>
       </div>
-      <div className="ten-year__wrapper">
```
Comment (BJHunnicutt) [inline packages/budget/src/components/CardCollection/index.js]: There are some styles in CardCollection.css for `.ten-year__wrapper`, are they needed? If not, want to remove them?

File: packages/budget/src/components/Budget101/index.js [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
 import DataTable from "./DataTable";
-import { BUDGET_101 } from "../constants";
 
 import "./Budget101.css";
 import "./stub.css";
 
 function Budget101() {
   return (
-    <StoryCard title="Budget 101" collectionId="budget" cardId={BUDGET_101}>
+    <CivicStoryCard watermark={<div />} footer={false} title="Budget 101">
```
Comment (BJHunnicutt) [inline packages/budget/src/components/Budget101/index.js]: This totally works, but I could see this `<div />` being confusing down the road. Would it make more sense to instead add an optional `hideWatermark` prop to CivicStoryCard?

**Comment by BJHunnicutt** [comment]
Ran it locally and everything seems to work for me! Just left a couple optional comments

File: packages/budget/src/components/CardCollection/index.js (hunk 1 above)
Comment (jaronheard) [reply packages/budget/src/components/CardCollection/index.js]: yep!

File: packages/budget/src/components/Budget101/index.js (hunk 2 above)
Comment (jaronheard) [reply packages/budget/src/components/Budget101/index.js]: I thought about this, and thought that leaving the `watermark` prop was more flexible without adding additional props to CivicStoryCard

**Pull request merged**